pkg/api/v1/schema.go

```diff
@@ -796,14 +796,16 @@ type Interface struct {
 	PciAddress string `json:"pciAddress,omitempty"`
 	// If specified the network interface will pass additional DHCP options to the VMI
 	// +optional
-	DhcpOptions `json:"dhcpOptions,omitempty"`
+	DHCPOptions `json:"dhcpOptions,omitempty"`
```

Comment:
That line should look like this:

```go
DHCPOptions *DHCPOptions `json:"dhcpOptions,omitempty"`
```

We don't want to embed the DHCPOptions struct, we want it a s a property.